docs/community/tutorials.md

```diff
@@ -0,0 +1,12 @@
+# Tutorials
+
+This page lists further learning materials and code walkthroughs for the H3 library and bindings. Contributions to this list are welcome, please feel free to open a [pull request](https://github.com/uber/h3/tree/master/docs/community/tutorials.md).
+
+## JavaScript
+
+- [H3 Tutorials on Observable](https://beta.observablehq.com/collection/@nrabinowitz/h3-tutorial)
+
+## Python
```

Comment:
if you want to add golang this is where I plan to host more examples: 
https://github.com/uber/h3-go/blob/master/example_test.go